File: sample-conference-videochat/sample-conference-videochat/Components/ChatScreen/Views/ChatCollectionView/ChatAttachmentIncomingCell/ChatAttachmentIncomingCell.m
//
//  ChatAttachmentIncomingCell.m
//  sample-conference-videochat
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "ChatAttachmentIncomingCell.h"
#import "UIView+Chat.h"

@interface ChatAttachmentIncomingCell()

@end

@implementation ChatAttachmentIncomingCell

- (void)awakeFromNib {
    [super awakeFromNib];

    [self.previewContainer  roundCornersWithRadius:6.0f isIncoming:YES];
}

+ (ChatCellLayoutModel)layoutModel {
    
    ChatCellLayoutModel defaultLayoutModel = [super layoutModel];
    defaultLayoutModel.avatarSize = CGSizeMake(40, 40);
    defaultLayoutModel.containerInsets = UIEdgeInsetsMake(0, 16, 0, 16);
    
    return defaultLayoutModel;
}

@end

File: sample-chat-swift/sample-chat-swift/Classes/Chat/CollectionView/CollectionCells/ChatAttachmentIncomingCell/ChatAttachmentIncomingCell.swift
//
//  ChatAttachmentIncomingCell.swift
//  Swift-ChatViewController
//
//  Created by Injoit on 1/28/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

import UIKit

class ChatAttachmentIncomingCell: ChatAttachmentCell {
    
    override class func layoutModel() -> ChatCellLayoutModel {
           var defaultLayoutModel = super.layoutModel()
           let containerInsets = UIEdgeInsets(top: 0.0, left: 16.0, bottom: 0.0, right: 16.0)
           defaultLayoutModel.containerInsets = containerInsets
           defaultLayoutModel.avatarSize = CGSize(width: 40.0, height: 40.0)
           
           return defaultLayoutModel
       }
}
